packages/relay/src/lib/eth.ts

```diff
@@ -147,7 +147,13 @@ export class EthImpl implements Eth {
   }
 
   private async getFeeWeibars() {
-    let networkFees = await this.mirrorNodeClient.getNetworkFees();
+    let networkFees;
+
+    try {
+      networkFees = await this.mirrorNodeClient.getNetworkFees();
+    } catch (e: any) {
+      this.logger.debug(`Failed to retrieve Network fees from Mirror Node`);
+    }
 
     if (_.isNil(networkFees)) {
       this.logger.debug(`Mirror Node returned no fees. Fallback to network`);
```

Comment:
This will log two statements for one error scenario, which is not ideal.

Reply:
Moved up so only 1 of the 2 logs show up